agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -152,12 +157,12 @@ function collect_ara_data {
 
 function collect_stockpile_data {
 	# parse config
-	stockpile_path=$(getconf.py stockpile_path stockpile)
-	stockpile_user=$(getconf.py stockpile_user stockpile)
-	stockpile_local_remote_user=$(getconf.py local_remote_user stockpile)
-	stockpile_host_remote_user=$(getconf.py host_remote_user stockpile)
-	stockpile_output_path=$(getconf.py stockpile_output_path stockpile)
-	stockpile_log=$(getconf.py stockpile_log stockpile)
+	stockpile_path=$(pbench-config stockpile_path stockpile)
+	stockpile_user=$(pbench-config stockpile_user stockpile)
+	stockpile_local_remote_user=$(pbench-config local_remote_user stockpile)
+	stockpile_host_remote_user=$(pbench-config host_remote_user stockpile)
+	stockpile_output_path=$(pbench-config stockpile_output_path stockpile)
+	stockpile_log=$(pbench-config stockpile_log stockpile)
```

Comment:
Have we switched to `pbench-config` in 0.69? I thought we only did that for 0.70+.

Reply:
Hmmm, I don't think we switched; I'll check and confirm here.

Reply:
Did you check this?

Reply:
Fixed.